scripts/configure.py

```diff
@@ -103,6 +103,12 @@
             'sudo systemctl enable systemd-journald.service',
             'sudo systemctl restart systemd-journald.service',
 
+            # Create directories and cronjobs for log persistence option
+            'sudo mkdir -p /var/log/journal',
+            'sudo systemd-tmpfiles --create --prefix /var/log/journal',
+            'sudo cp scripts/delete_persisted_logs.sh /usr/local/bin',
+            'sudo cp config/delete_persisted_logs_crontab /etc/cron.d/delete_persisted_logs',
```

Comment:
Separated this chunk into its own section for the final edition of this code rather than having them mixed in under the other sectioned chunks within this logging-handling script